Assets/Shared/Scripts/GameManager.cs

```diff
@@ -1,14 +1,16 @@
-﻿using System.Collections;
+﻿using System;
+using System.Collections;
 using System.Collections.Generic;
+using System.Linq;
 using UnityEngine;
 
 public class GameManager : MonoBehaviour {
 
 	public static GameManager instance = null;
 
 	public bool paused = false;
-	public GameObject character;
-	public Transform spawnPoint;
+	public Dictionary <CharacterInfo, CharacterName> characters = new Dictionary <CharacterInfo, CharacterName>();
```

Comment:
Nah, the first Type in the dictionary is the key, the second is the value. Seems to me that the CharacterName needs to be the key and the CharacterInfo the value.

Reply:
Ergo, you can get the right CharacterInfo object by using characters[CharacterName.Ness], etc.